recipes/gtest/all/conandata.yml

```diff
@@ -2,3 +2,10 @@ sources:
   "1.12.1":
     url: "https://github.com/google/googletest/archive/release-1.12.1.tar.gz"
     sha256: "81964fe578e9bd7c94dfdb09c8e4d6e6759e19967e397dbea48d1c10e45d0df2"
+  "1.10.0":
+    url: "https://github.com/google/googletest/archive/release-1.10.0.tar.gz"
+    sha256: "9dc9157a9a1551ec7a7e43daea9a694a0bb5fb8bec81235d8a1e6ef64c716dcb"
+patches:
+  "1.10.0":
+    - patch_file: "patches/gtest-1.10.0.patch"
```

Comment:
Please, add patch_type and patch_description at least. It would be nice adding patch_source too if possible.

https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches

Reply:
@uilianries Thank you a lot for your comments.
fixed.